src/main/scala/algolia/AlgoliaClient.scala

```diff
@@ -94,8 +93,14 @@ class AlgoliaClient(
 
   val analyticsHost: String = "https://analytics.algolia.com"
   val insightsHost: String = "https://insights.algolia.io"
+
   /* Recommendation default host is set as 'var' because the region might be overridden. */
-  var recommendationHost: String = "https://recommendation.us.algolia.com"
+  @deprecated("use personalizationHost instead", "1.40.0")
+  var recommendationHost: String = "https://personalization.us.algolia.com"
+  def personalizationHost: String = recommendationHost
+  def personalizationHost(host: String) = {
+    recommendationHost = host
+  }
```

Comment:
```suggestion
  var personalizationHost: String = "https://personalization.us.algolia.com"
  def personalizationHost(host: String) = {
    recommendationHost = host
  }
  @deprecated("use personalizationHost instead", "1.40.0")
  def recommendationHost: String = personalizationHost
```

I don't know what the difference between var and def is, but deprecated code shouldn't be used in our own code, but the deprecated code should use the undeprecated version

Reply:
After a second though, I agree! I've separated the two 👍